### PR TITLE
Add rna cd recipe

### PR DESCRIPTION
**Pull request by sndrtj**
:information_source:
Bioconda has finished the [GCC7 migration](https://github.com/bioconda/bioconda-recipes/issues/13578). If you are dealing with C/C++ or Python package it can be that you need to rebuild other dependent packages. [Bioconda utils - update-pinning](https://bioconda.github.io/updating.html#updating-recipes-for-a-pinning-change) will assist you with that. If you have any questions please use [issue 13578](https://github.com/bioconda/bioconda-recipes/issues/13578).

----------------

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by sndrtj** [comment]
@bioconda/core could you review this package? thanks!

**Comment by dpryan79** [comment]
Sure, I'll take a look. Note sure why pinging didn't work for /core.

File: recipes/rna-cd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  license: GNU Affero General Public v3 or later (AGPLv3+)
+  license_family: AGPL
+  license_file: 
+  summary: RNA contamination detector
+  doc_url: https://rna-cd.readthedocs.io/en/latest/ 
+  dev_url: https://github.com/LUMC/rna_cd
+
+extra:
+  recipe-maintainers:
+    - sndrtj
```
Comment (dpryan79) [inline recipes/rna-cd/meta.yaml]: Can you remove the empty `license_file:` line? You can also remove the `extra:` section if you like, we don't actually use it for anything (only conda-forge does). Other than that it looks good!

Comment (sndrtj) [reply recipes/rna-cd/meta.yaml]: Thanks for the review! This is done in commit 80f1edb.

**Comment by dpryan79** [comment]
:+1: Looks great, thanks!

**Pull request merged**